catalog/dags/providers/provider_api_scripts/cc_mixter.py

```diff
@@ -0,0 +1,274 @@
+"""
+Content Provider:       ccMixter
+
+ETL Process:            Use the API to identify all CC licensed media.
+
+Output:                 TSV file containing the media and the
+                        respective meta-data.
+
+Notes:                  Documentation: https://ccmixter.org/query-api
+                        ccMixter sends bad JSON and extremely huge headers, both
+                        of which need workarounds that are handled by this DAG.
+"""
+
+import json
+import logging
+import re
+from typing import Literal
+
+from common import constants
+from common.licenses import get_license_info
+from common.loader import provider_details as prov
+from common.requester import DelayedRequester
+from providers.provider_api_scripts.provider_data_ingester import ProviderDataIngester
+
+
+logger = logging.getLogger(__name__)
+
+JSON_OCTALS = re.compile(r":\s*0(?P<num>\d+)\s*(?P<sep>[,}])")
+
+
+class CcMixterDelayedRequester(DelayedRequester):
+    """
+    WORKAROUND!
+
+    ccMixter sends bad JSON, including numbers with a leading 0 (observed some
+    such cases with "bpm" field). This makes the JSON invalid and raises decode
+    errors.
+
+    This class extends ``DelayedRequester`` to supply a custom JSON decoding
+    step where we perform a text substitution first and then parse the JSON.
+    """
+
+    def _get_json(self, response):
+        raw_json = response.text
+        cleaned_json = JSON_OCTALS.sub(r":\g<num>\g<sep>", raw_json)
+        if cleaned_json == raw_json:
+            logger.info("JSON was clean, no substitutions were made.")
```

Comment:
Perhaps this will be better as a `debug` call so logs are not so long.
```suggestion
            logger.debug("JSON was clean, no substitutions were made.")
```